data/db/slr-canon.xml

```diff
@@ -4665,6 +4676,28 @@
 
     <lens>
         <maker>Canon</maker>
+        <model>Canon EF 300mm f/4L IS</model>
+        <mount>Canon EF</mount>
+        <cropfactor>1.605</cropfactor>
+        <calibration>
+            <distortion model="ptlens" focal="300" a="-0.00271" b="0.00945" c="-0.00645" />
+            <tca model="poly3" focal="300" br="0.0000114" vr="0.9999147" bb="-0.0000121" vb="0.9999900" />
+        </calibration>
+    </lens>
+
+    <lens>
+        <maker>Canon</maker>
+        <model>EF300mm f/4L IS USM +1.4x</model>
```

Comment:
Thanks, that's what I was hoping for feedback on. I thought I'd read through the calibration document, but clearly not well enough, sorry. I'll add the camera comments as well.

A Lot of the similar calibrations have translations for the converters as well, is it useful to copy that here?